src/System.CommandLine.Tests/System.CommandLine.Tests.csproj

```diff
@@ -32,6 +32,8 @@
     <Compile Include="..\Common\ArgumentBuilder.cs" Link="Utility\ArgumentBuilder.cs" />
     <Compile Include="..\Common\OptionBuilder.cs" Link="Utility\OptionBuilder.cs" />
     <Compile Include="..\System.CommandLine.Suggest\DotnetMuxer.cs" Link="Utility\DotnetMuxer.cs" />
+    <Compile Include="..\System.CommandLine\LocalizationResources.cs" Link="LocalizationResources.cs" />
```

Comment:
I am including these types and the resources here so the tests can call the internal APIs to verify error messages.